core/src/components/radio/radio.md.vars.scss

```diff
@@ -11,7 +11,7 @@ $radio-md-color-on:                  current-color(base) !default;
 $radio-md-background-color-focused:  ion-color(primary, tint) !default;
 
 /// @prop - Color of the unchecked radio
-$radio-md-color-off:                 $text-color-step-600 !default;
+$radio-md-color-off:                 rgb($text-color-rgb, 0.60) !default;
```

Comment:
It looks like Material Design Components for the Web use 0.54 instead of 0.6: https://material-components.github.io/material-components-web-catalog/#/component/radio. However, I do see that the Figma file uses 0.6. Which one is correct?

![image](https://github.com/ionic-team/ionic-framework/assets/2721089/31acfb72-ed35-460a-b32e-b2e05e82cb3a)

Reply:
I think the web catalog is outdated. This opacity seems to follow the Material Design 1 opacity for [checkbox](https://m1.material.io/components/selection-controls.html#selection-controls-checkbox) and [radio](https://m1.material.io/components/selection-controls.html#selection-controls-radio-button):

<img width="1241" alt="Screenshot 2023-05-10 at 11 50 03 AM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/55bd9c76-df9e-4b86-ace1-2c431973d879">
<img width="1232" alt="Screenshot 2023-05-10 at 11 52 00 AM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/3270d22b-d833-452b-b3cb-71612ab3af95">

I would vote we follow the Figma since there is also a [Material Design 3 kit](https://www.figma.com/community/file/1035203688168086460) available.

Reply:
Makes sense. Thanks!